crates/router/src/compatibility/stripe/app.rs

```diff
@@ -88,7 +88,7 @@ impl Customers {
                     .route(web::delete().to(customer_delete)),
             )
             .service(
-                web::resource("/{customer_id}/payment_methods")
+                web::resource("/payment_methods")
```

Comment:
Why was this changed. It doesn't match stripe endpoint now https://stripe.com/docs/api/payment_methods/customer_list

Reply:
done, fixed it!